integration-tests/bats/status.bats

```diff
@@ -453,3 +456,35 @@ SQL
     mv .dolt/repo_state.backup .dolt/repo_state.json
     [ "$status" -eq 0 ]
 }
+
+@test "status: ignored tables show up" {
+    dolt sql <<SQL
+CREATE TABLE t (pk int PRIMARY KEY);
```

Comment:
can you use identifiers which are 3 chars or more? Easier for future maintainers to search/replace. I know we do single char vars all over the place - just trying to raise the bar one change at a time :-)

Reply:
Using (slightly) more descriptive identifiers.

Reply:
Looks like you changed them in sql-local-remote.bats, but not in status.bats. I'm not going to block on this. Do as you see fit.